src/main/kotlin/app/cash/licensee/plugin.kt

```diff
@@ -143,7 +145,7 @@ private fun configureAndroidVariants(
 ) {
   val androidComponents = project.extensions.getByType(AndroidComponentsExtension::class.java)
   androidComponents.onVariants { variant ->
-    val suffix = variant.name.capitalize(ROOT)
+    val suffix = variant.name.replaceFirstChar { it.titlecase(ROOT) }
```

Comment:
Don't need to do the `if (it.isLowerCase())` dance that they have in their deprecation docs?

Reply:
In this case it's always already going to be lowercase